Hazel/src/Hazel/Renderer/Renderer2D.cpp

```diff
@@ -341,4 +343,4 @@ namespace Hazel {
 		return s_Data.Stats;
 	}
 
-}
```

Comment:
Why did this move?

Reply:
These are just file endings... not really something I give much attention to.. Maybe we should see to get #52 integrated, because this is not the first time file endings change unneeded ...

Reply:
Yeah, we definitely need #52 integrated.